Tests/SilicaTests/Utilities/HTTP/HTTPClient.swift

```diff
@@ -7,6 +7,7 @@
 //
 
 import Foundation
+import FoundationNetworking
```

Comment:
Did you try compiling this on macOS? This is a Linux-only module, it should be guarded with `#if canImport()`

Reply:
Added and tested on Linux and MacOS